lib/models/message.ts

```diff
@@ -281,4 +293,52 @@ export class MessageModel extends DocumentDbModel<
       query: "SELECT * FROM messages m WHERE (m.fiscalCode = @fiscalCode)"
     });
   }
+
+  public async attachStoredContent(
+    blobService: BlobService,
+    messageId: string,
+    partitionKey: string,
+    message: IMessageWithContent
+  ): Promise<Either<Error, Option<DocumentDb.AttachmentMeta>>> {
+    // this is the attachment id __and__ the filename
```

Comment:
comment is not very clear, I believe is safe to inline `getMessageAttachmentName` here since it's called only from here

Reply:
ok